packages/stable/src/types.ts

```diff
@@ -571,30 +571,37 @@ export interface DataSetChangeByExternalId extends DataSetPatch, ExternalId {}
 
 export interface DataSetChangeById extends DataSetPatch, InternalId {}
 
-export type Label = ExternalId;
+export type AttachedLabel = ExternalId;
```

Comment:
Now this looks even worse, I would suggest to get rid of it and use external id instead :)